integrations/huggingface-transformers/tutorials/sparse-transfer-learning-bert.md

```diff
@@ -0,0 +1,343 @@
+
+Here is what the recipe looks like:
+
+```yaml
+version: 1.1.0
+
+# General Variables
+num_epochs: &num_epochs 13
+init_lr: 1.5e-4
+final_lr: 0
+
+qat_start_epoch: &qat_start_epoch 8.0
+observer_epoch: &observer_epoch 12.0
+quantize_embeddings: &quantize_embeddings 1
+
+distill_hardness: &distill_hardness 1.0
+distill_temperature: &distill_temperature 2.0
+
+weight_decay: 0.01
+
+# Modifiers:
+training_modifiers:
+  - !EpochRangeModifier
+      end_epoch: eval(num_epochs)
+      start_epoch: 0.0
+
+  - !LearningRateFunctionModifier
+      start_epoch: 0
+      end_epoch: eval(num_epochs)
+      lr_func: linear
+      init_lr: eval(init_lr)
+      final_lr: eval(final_lr)
+
+quantization_modifiers:
+  - !QuantizationModifier
+      start_epoch: eval(qat_start_epoch)
+      disable_quantization_observer_epoch: eval(observer_epoch)
+      freeze_bn_stats_epoch: eval(observer_epoch)
+      quantize_embeddings: eval(quantize_embeddings)
+      quantize_linear_activations: 0
+      exclude_module_types: ['LayerNorm', 'Tanh']
+      submodules:
+        - bert.embeddings
+        - bert.encoder
+        - bert.pooler
+        - classifier
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+
+regularization_modifiers:
+  - !SetWeightDecayModifier
+      start_epoch: 0.0
+      weight_decay: eval(weight_decay)
+```
+   
+The key `Modifiers` for sparse transfer learning are the following:
+- `ConstantPruningModifier` instructs SparseML to maintain the sparsity structure of the network during the fine-tuning process
+- `QuantizationModifier` instructs SparseML to apply quantization aware training to quantize the weights over the final epochs
+- `DistillationModifier` instructs SparseML to apply model distillation at the logit layer
+   
+As a result, we end up with a 95% pruned and quantized BERT trained on SST2! It achieves ~92% accuracy.
+
+#### Aside: Dense Teacher Creation (OPTIONAL)
+
+By passing a `--distill_teacher` argument, we instruct SparseML to apply model distillation during the fine-tuning process. This is optional, but passing a dense teacher can help improve accuracy.
+
+Above, we instructed SparseML to download the dense teacher from SparseZoo. You can also pass a path to a directory containing a your transformers model and configuration.
```

Comment:
You can also pass a path to a directory containing ~~a~~ your transformers model and configuration.